root/inc/notification.html

```diff
@@ -4,38 +4,36 @@
     MAILTO = format('mailto:%s?subject=%s&body=%s');
     NOTIFICATION = {
         NEEDHELP = {
-            has_email = 1,
             title = "Looking for help!",
             text = "The maintainer of this distribution is looking for people to help them improve this module! If you're interested then please contact them via ",
-            body = format("Hey %s,\r\n\r\nI'm willing to provide some time to help you with new feature for the %s module. if you can let me know what you would like to be done.\r\n\r\nKind Regards,")
+            body = format("Hello,\r\n\r\nI'm willing to provide some time to help you with new feature for the %s module, if you can let me know what you would like to be done.\r\n\r\nKind Regards,")
         },
         ADOPTME = {
-            has_email  = 1,
             title  = "Why not adopt me?",
             text   = "This distribution is up for adoption! If you're interested then please contact the PAUSE module admins via ",
-            body   = format("Hey %s,\r\n\r\nI'm interested in adopting the %s module. If you can grant me the privilege I will release the next version.\r\n\r\n Kind Regards,"),
+            body   = format("Hello,\r\n\r\nI'm interested in adopting the %s module. If you can grant me the privilege I will release the next version.\r\n\r\n Kind Regards,"),
             email  = 'modules@perl.org',
             name   = 'PAUSE Admins'
         },
         HANDOFF = {
-            has_email  = 1,
             title  = "Take me over?",
             text   = "The maintainer of this distribution is looking for someone to take over! If you're interested then please contact them via ",
-            body   = format("Hey %s,\r\n\r\nI'm interested in taking over the %s module. If you can grant me the privilege I will release the next version.\r\n\r\n Kind Regards,")
+            body   = format("Hello,\r\n\r\nI'm interested in taking over the %s module. If you can grant me the privilege I will release the next version.\r\n\r\n Kind Regards,")
         },
     }.$notification;
+    IF NOTIFICATION.email;
+        notifiable.push(NOTIFICATION.email);
+    END;
     IF pod_file && pod_file.module.0;
-        NOTIFICATION.module = pod_file.module.0.name;
+        NOTIFICATION.module = pod_file.module.0.name || release.main_module;
```

Comment:
This should be pulling from the notification hash, as described in my other comment.

Reply:
Now implemented